test/keyboard.spec.ts

```diff
@@ -204,35 +208,35 @@ describe('Keyboard', function() {
           'Keyup: # Digit3 51 [Shift]',
           'Keyup: Shift ShiftLeft 16 []'].join('\n'));
   });
-  it('should specify repeat property', async({page, server}) => {
+  it('should specify repeat property', async ({page, server}) => {
     await page.goto(server.PREFIX + '/input/textarea.html');
     await page.focus('textarea');
-    await page.evaluate(() => document.querySelector('textarea').addEventListener('keydown', e => window.lastEvent = e, true));
+    await page.evaluate(() => document.querySelector('textarea').addEventListener('keydown', e => lastEvent = e, true));
     await page.keyboard.down('a');
-    expect(await page.evaluate(() => window.lastEvent.repeat)).toBe(false);
+    expect(await page.evaluate(() => lastEvent.repeat)).toBe(false);
     await page.keyboard.press('a');
-    expect(await page.evaluate(() => window.lastEvent.repeat)).toBe(true);
+    expect(await page.evaluate(() => lastEvent.repeat)).toBe(true);
 
     await page.keyboard.down('b');
-    expect(await page.evaluate(() => window.lastEvent.repeat)).toBe(false);
+    expect(await page.evaluate(() => lastEvent.repeat)).toBe(false);
     await page.keyboard.down('b');
-    expect(await page.evaluate(() => window.lastEvent.repeat)).toBe(true);
+    expect(await page.evaluate(() => lastEvent.repeat)).toBe(true);
 
     await page.keyboard.up('a');
     await page.keyboard.down('a');
-    expect(await page.evaluate(() => window.lastEvent.repeat)).toBe(false);
+    expect(await page.evaluate(() => lastEvent.repeat)).toBe(false);
   });
-  it('should type all kinds of characters', async({page, server}) => {
+  it('should type all kinds of characters', async ({page, server}) => {
     await page.goto(server.PREFIX + '/input/textarea.html');
     await page.focus('textarea');
     const text = 'This text goes onto two lines.\nThis character is 嗨.';
     await page.keyboard.type(text);
     expect(await page.evaluate('result')).toBe(text);
   });
-  it('should specify location', async({page, server}) => {
+  it('should specify location', async ({page, server}) => {
     await page.goto(server.PREFIX + '/input/textarea.html');
     await page.evaluate(() => {
-      window.addEventListener('keydown', event => window.keyLocation = event.location, true);
+      window.addEventListener('keydown', event => window['keyLocation'] = event.location, true);
```

Comment:
nit: IIUC, this change is not needed

Reply:
These and below: TypeScript treats `window.somevar` differently from `window['somevar']`.